examples/decoding/plot_decoding_ssd_csp_eeg.py

```diff
@@ -0,0 +1,256 @@
+"""
+.. _ex-decoding-ssd-csp-eeg:
+
+============================================================================
+Improving motor imagery decoding from EEG using Spatio Spectra Decomposition
+============================================================================
+
+Improving the decoding of Motor Imagery from EEG data when SSD is applied
```

Comment:
SSD and CSP both need to be defined before acronyms are used. SSD is in the title, but it should be in the text too. And CSP does not seem defined anywhere.

Reply:
I agree - and a short motivation of why SSD would work for this could be added as well (it gets explained at the end of the example, but would be nice to motivate at the beginning already).

Reply:
Sure! I totally missed that. Please, check how does it look now.